src/vaadin-contextmenu-items-mixin.html

```diff
@@ -186,6 +191,7 @@
       }}));
     }
 
+    /** @private */
     __itemsRenderer(root, menu, context) {
```

Comment:
This is only used in `vaadin-context-menu.html`. Should be `@protected`?

Reply:
Done.